includes/admin/wc-meta-box-functions.php

```diff
@@ -181,6 +181,33 @@ function woocommerce_wp_checkbox( $field ) {
 	echo '</p>';
 }
 
+
+/**
+ * Return the html selected attribute if stringified $value is found in array of stringified $options
+ * or if stringified $value is the same as scalar stringified $options.
+ *
+ * @param string|int       $value   Value to find within options.
+ * @param string|int|array $options Options to go through when looking for value.
+ * @return string
+ */
+function wc_selected( $value, $options ) {
```

Comment:
You can resume this functions to:

```php
function wc_selected( $value, $options ) {
	if ( is_array( $options ) ) {
		$options = array_map( 'strval', $options );
		return selected( in_array( (string) $value, $options, true ), true, false );
	}

	return selected( $value, $options, false );
}
```

So you can ignored the next comment about `selected='selected'`